src/main/python/rlbot/utils/virtual_environment_management.py

```diff
@@ -28,8 +59,9 @@ def post_setup(self, context: SimpleNamespace) -> None:
         sys.stderr.write(f'Installing {requirements}...\n')
         sys.stderr.flush()
 
-        args = [context.env_exe, '-m', 'ensurepip']
-        finished_process = self.run_and_dump(args, timeout=120)
+        # Actually update pip
+        args = [context.env_exe, '-m', 'pip', 'install', '-U', 'pip']
+        finished_process = self.run_and_dump(args, timeout=300)
```

Comment:
What's the story on this? Upgrading pip seems like a good idea, but is it safe to remove `env_exe -m ensurepip`? Are we guaranteed to always already have pip at this point?

Reply:
Yes, with_pip guarantees that we will.